src/components/drop-down-menu/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import protoTypes from 'prop-types'
+import classnames from 'classnames'
+import { IoMdArrowDropdown } from "react-icons/io";
+import {
+    Menu,
+    MenuItems,
+    MenuButton,
+    MenuItem,
+    MenuPopover,
+} from "@reach/menu-button";
+
+const DropDownMenu = (props) => {
+    const {
+        id,
+        title,
+        menuList,
+        className,
+        ...compProps
+    } = props;
+
+    const getToggleSvg = (isExpanded) => {
+        return (
+            <div className={classnames('transform transition-all duration-500', {
+                'rotate-0': !isExpanded,
+                '-rotate-180': isExpanded
+            })}>
+                <IoMdArrowDropdown size={24} />
+            </div>
+        );
+    }
+
+    return (
+        <div id={id} className={className} {...compProps}>
+            <Menu className='relative'>
+                {({ isExpanded }) => (
+                    <React.Fragment>
```

Comment:
Can't we normal fragments 
    **<>{Code}</>**

Reply:
done